.spelling

```diff
@@ -3,48 +3,73 @@
 # global dictionary is at the start, file overrides afterwards
 # one word per line, to define a file override use ' - filename'
 # where filename is relative to this configuration file
+#region Global Dictionary
```

Comment:
Does VSCode support collapsing these regions?  I think from a readability standpoint, using whitespace between sections may be easier to read rather than decorating them with `#region` unless there's an editor benefit.

Reply:
User defined region folding is in the [iteration plan](https://github.com/Microsoft/vscode/issues/34160) for the 29th, that's why I included it. 😀 I could do white space too, though.

Reply:
My personal preference is to go with whitespace for now.  We can always add regions later.  This is not a blocking issue for me, however.